test/message/test_plan.py

```diff
@@ -473,62 +643,6 @@ def test__replace_message_ids():
     assert replaced == expected
 
 
-def test___call__function(hook):
```

Comment:
there's no way to reuse these tests?

Reply:
I don't know really what to do with these tests, I can hardly them fit with the new notions of stateful plans :/

Reply:
I see, that's okay, I guess. :disappointed: